src/ServicePulse.Host/app/js/views/endpoint_details/endpoint_details.html

```diff
@@ -36,28 +36,28 @@
             </div>
 
             <div class="row">
-                <div class="col-sm-6 metric-digest"><span class="metric-digest-header">Queue Length</span><span class="metric-digest-value">{{endpoint.digest.metrics.queueLength.latest | number:0}} msgs</span></div>
-                <div class="col-sm-6 metric-digest"><span class="metric-digest-header">Queue Length</span><span class="metric-digest-value">{{endpoint.digest.metrics.queueLength.average | number:0}} msgs</span></div>
+                <div class="col-sm-6 metric-digest"><span class="metric-digest-header" uib-tooltip="Queue length (# msgs): The estimated number of messages in an endpoint queue.">Queue Length</span><span class="metric-digest-value">{{endpoint.digest.metrics.queueLength.latest | number:0}} msgs</span></div>
```

Comment:
This should probably be:

`The estimated number of messages in an endpoint's queue.`  (It's missing the possessive apostrophe)

Reply:
To answer this question I will send my champion :P @adamralph @tmasternak 
You were the authors of this words, could you respond to @WilliamBZA 's (possessive apostrophe ;)) comment

Reply:
The phrase is currently "endpoint queue" (no possessive modification), but @WilliamBZA's suggestion of "endpoint's queue" is probably better.